api/v1/handlers.go

```diff
@@ -2,24 +2,11 @@ package api
 
```

Comment:
rename this file from `handlers.go` to `status.go` now that its only doing one thing